### ISSUE TITLE
Accessibility Issue in p-confirmdialog

### ISSUE THREAD

**Issue by justinthomas0789** [primefaces/primeng#18163]
Hi Team,
After updating to PrimeNG v19, I noticed a key issue with the p-confirmdialog component:

**Issue: Accessibility Issue with the Close Button**
The close button in p-confirmdialog does not have an aria-label or accessible text by default. While the documentation mentions the closeAriaLabel property, setting it does not apply the aria-label to the close button.

**Example:**

<p-confirmdialog closeAriaLabel="{{ 'FORM.CLOSE' | translate }}"></p-confirmdialog>

**Expected Behavior:**

The aria-label should appear on the close button, ensuring accessibility compliance (e.g., WCAG standards).

**Actual Behavior:**

The aria-label is missing from the close button, resulting in accessibility flags during audits.

I created an issue before regarding this; the ticket is closed but it seems the issue is still there. link follows
[https://github.com/primefaces/primeng/issues/17404](https://github.com/primefaces/primeng/issues/17404)

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/5fqkywvc?file=src%2Fapp%2Fconfirm-dialog-basic-demo.html

### Environment

Angular v19
Primeng v 19

### Angular version

19.2.8

### PrimeNG version

v19

### Browser(s)

Chrome latest

### Steps to reproduce the behavior

See the above example, inspect the close button

### Expected behavior

There should be an aria-label attribute in the button
